openedx/core/djangoapps/content/course_overviews/migrations/0010_auto_20160329_2317.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_overviews', '0009_readd_facebook_url'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='courseoverview',
+            name='facebook_url',
+        ),
```

Comment:
Correct, the original PR had migration code in it to remove that field from the table without previously having removed it from the model, which would have broken production.  That RemoveField calls was removed.  You can look at 0008 and 0009 for details.

Now that we've removed the model field in a release, it should be safe to remove the column in another release. :+1: from me, provided we've done the usual work when migrating this table to deal with row versions.